stable/mcrouter/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+name: mcrouter
+version: 0.1.0
+engine: gotpl
```

Comment:
Can you add the metadata that is missing from here:
https://github.com/kubernetes/charts/blob/master/stable/jenkins/Chart.yaml

Namely:
- home
- appVersion (the version of mcrouter you are using)
- description
- sources (where is the source code for the docker images and mcrouter itself)
- maintainers
- icon (is there an icon for mcrouter?)